Iniciante/1114/README.md

```diff
@@ -0,0 +1,40 @@
+# Problema:
+
+Escreva um programa que repita a leitura de uma senha até que ela seja válida. Para cada leitura de senha incorreta informada, escrever a mensagem "Senha Invalida". Quando a senha for informada corretamente deve ser impressa a mensagem "Acesso Permitido" e o algoritmo encerrado. Considere que a senha correta é o valor 2002.
+
+##### Problema completo: https://www.urionlinejudge.com.br/judge/pt/problems/view/1114
+
+# Resolução:
+
+O objetivo desse exercício é verificar se o valor lido é 2002 e mostrar na tela a mensagem `Acesso Permitido` se o valor for 2002 ou `Senha Invalida` se não for. 
+
+Nesse exercício só precisamos de uma variável do tipo `int`:
+```c
+    int senha;
+```
+`senha` é a variável que será lida e comparada com o número 2002, valor dado no enunciado.
+
+Precisamos ler a senha inúmeras vezes, já que não sabemos quantas vezes o usuário vai errar a senha. Aqui vamos usar o `do.. while`:
```

Comment:
A frase "Precisamos ler a senha inúmeras vezes, já que não sabemos quantas vezes o usuário vai errar a senha" não é correta, pois não é garantido que precisamos ler a senha inúmeras vezes e nem a justificativa utilizada, pois não saber quantas vezes o usuário vai errar a senha não quer dizer nem um nem outro, então não se pode tomar uma certeza arredor disso.
 
Uma sugestão seria algo do tipo, como precisamos de uma estrutura que efetue a leitura enquanto a senha correta não for lida ...  para isso utilizaremos o do.. while

Reply:
acho q seria bem valido explicar como funciona o do while, q primeiro ele executa uma vez e dps verifica , até para ajudar na explicação da linha 30